bin/preplinters.sh

```diff
@@ -4,10 +4,9 @@
 ## the various linters that are used over the
 ## mixer source base.
 
-set -ev
-
 go get -u github.com/alecthomas/gometalinter
 go get -u github.com/bazelbuild/buildifier/buildifier
 go get -u github.com/3rf/codecoroner
 gometalinter --install >/dev/null
 bin/bazel_to_go.py
+ln -s -f bazel-genfiles/adapter/denyChecker adapter/denyChecker/config_proto
```

Comment:
Yeah, this is not a great story. Maybe Mandar can figure out a more generic approach in his Python script.